common/jenkins-agents/terraform-2408/docker/Dockerfile.ubi8

```diff
@@ -0,0 +1,211 @@
+FROM opendevstackorg/ods-jenkins-agent-base-ubi8:latest
+
+LABEL maintainer="Erhard Wais <erhard.wais@boehringer-ingelheim.com>, Frank Joas <frank.joas@boehringer-ingelheim.com>, Josef Hartmann <josef.hartmann@boehringer-ingelheim.com>, Steve Taylor <steve.taylor@boehringer-ingelheim.com>"
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-agent-terraform-ubi8-docker" \
+      name="openshift/jenkins-agent-terraform-ubi8" \
+      version="0.1" \
+      architecture="x86_64" \
+      release="1" \
+      io.k8s.display-name="Jenkins Agent Terraform" \
+      io.k8s.description="The jenkins agent image has terraform and other tools on top of the jenkins agent base image." \
+      io.openshift.tags="openshift,jenkins,agent,terraform,aws,azure"
+
+ENV TERRAFORM_VERSION=1.9.4 \
+    TERRAFORM_CONFIG_INSPECT_VERSION=0.2.0 \
+    TERRAFORM_DOCS_VERSION=v0.18.0 \
+    RUBY_VERSION=3.3.4 \
+    PACKER_VERSION=1.11.2 \
+    CONSUL_VERSION=1.19.1 \
+    TENV_VERSION=3.0.0 \
+    TENV_AUTO_INSTALL=true \
+    TENV_ROOT=/opt/tenv \
+    GO_VERSION=1.21.13 \
+    GOBIN=/usr/local/go/bin \
+    TFLINT_VERSION=0.52.0 \
+    NODEJS_VERSION=20.16.0 \
+    BUNDLER_VERSION=2.5.17 \
+    SOPS_VERSION=3.9.0 \
```

Comment:
The installation of SOPS could be removed as it is already installed in the base agent -> https://github.com/opendevstack/ods-core/blob/master/jenkins/agent-base/Dockerfile.ubi8#L76

Reply:
good catch - thx for the hint

Reply:
removed